.lando.yml

```
@@ -43,9 +43,17 @@ services:
       - BRANCH_NAME=$(git -C /usr/local/quickstart-install-profile rev-parse --abbrev-ref HEAD); if [ $(git ls-remote --heads https://github.com/az-digital/az-quickstart-scaffolding.git $BRANCH_NAME | wc -l) = 1 ]; then SCAFFOLD_BRANCH="$BRANCH_NAME"; else SCAFFOLD_BRANCH="main"; fi; git clone --branch $SCAFFOLD_BRANCH https://github.com/az-digital/az-quickstart-scaffolding.git /app
       - composer config repositories.localdev path /usr/local/quickstart-install-profile && composer require --no-update az-digital/az_quickstart:\*@dev
       # Use the matching require-dev metapackage branch if available.
-      - BRANCH_NAME=$(git -C /usr/local/quickstart-install-profile rev-parse --abbrev-ref HEAD); if [ $(git ls-remote --heads https://github.com/az-digital/az-quickstart-dev.git $BRANCH_NAME | wc -l) = 1 ]; then DEV_PACKAGE_BRANCH="$BRANCH_NAME"; else DEV_PACKAGE_BRANCH=main; fi
-      - if [[ ${DEV_PACKAGE_BRANCH} =~ ^[0-9]\.[0-9x] ]]; then DEV_PACKAGE_VERSION="${DEV_PACKAGE_BRANCH}-dev"; else DEV_PACKAGE_VERSION="dev-${DEV_PACKAGE_BRANCH}"; fi
-      - composer require --no-update --dev az-digital/az-quickstart-dev:dev-${DEV_PACKAGE_VERSION}
+      - |
+        BRANCH_NAME=$(git -C /usr/local/quickstart-install-profile rev-parse --abbrev-ref HEAD)
+        if [ $(git ls-remote --heads https://github.com/az-digital/az-quickstart-dev.git $BRANCH_NAME | wc -l) = 1 ]; then
+          DEV_PACKAGE_BRANCH="$BRANCH_NAME"; else DEV_PACKAGE_BRANCH=main
```

Use more consistent whitespace + we should actually be defaulting to `2.x` in the `2.x` version of this file.
```suggestion
          DEV_PACKAGE_BRANCH="$BRANCH_NAME"
        else DEV_PACKAGE_BRANCH=2.x
```